Dockerfile.prod

```diff
@@ -32,6 +33,8 @@ RUN apk add curl
 
 USER node
 COPY --chown=node:node --from=build /app /app
+# Add this the service Dockerfile, at the final stage if multi-stage
+COPY --from=docker.atl-paas.net/sox/brahmos-deps/stress-ng:latest /usr/bin/stress-ng /usr/bin/stress-ng
```

Comment:
Was wondering if it is safe to put it to prod Dockerfile though...

Reply:
@bgvozdev as i understood we do not have a separate file for stg image. I think that it shouldn't be a problem, we just will not have a brahmos on a prod instance